### PR TITLE
Improve this role

### PR DESCRIPTION
**Pull request by genofire**
sorry for the bunch on commit:
- more configurable `bbb_mute_on_start`, `bbb_app_log_level` and  `bbb_meteor` (like audio skipCheck and camera quality)
- make multiple stun/turn-server configurable
- fix ci (hopefully)
- drop debug messages #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,25 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
...
 | Variable Name | Function | Default value | Comment |
 | ------------- | -------- | ------------- | ------- |
-| `bbb_hostname` | Hostname for this BigBlueButton instance _(required)_ | {{ ansible_fqdn_hostname }} |
+| `bbb_hostname` | Hostname for this BigBlueButton instance _(required)_ | `{{ ansible_fqdn_hostname }}` |
 | `bbb_letsencrypt_enable` | Enable letsencrypt/HTTPS | `yes` |
 | `bbb_letsencrypt_email` | E-mail for use with letsencrypt _(required when using LE)_|  |
 | `bbb_coturn_enable` | enable installation of the TURN-server | `yes` |
+| `bbb_coturn_port` | the port for the TURN-Server to use | `3443` | 
+| `bbb_coturn_port_tls` | the port for tls for the TURN-Server to use | `3443` | 
+| `bbb_coturn_secret` | Secret for the TURN-Server  _(required)_ | | can be generated with `openssl rand -hex 16`
```
Comment (n0emis) [inline README.md:13]: Since this is a somewhat breaking change, please also change the example-playbook and the tests to reflect this...

Comment (genofire) [reply README.md:15]: we will see, if i make it right now ;)

File: README.md [hunk 2]
```diff
@@ -4,22 +4,29 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
...
+| `bbb_turn_servers` | a list of TURN-Server to use | `{{ bbb_hostname }}` with `{{ bbb_coturn_secret }}` | take a look in defaults/main.yml
 | `bbb_greenlight_enable` | enable installation of the greenlight client | `yes` |
 | `bbb_greenlight_hosts` | the hostname that greenlight is accessible from | `{{ bbb_hostname }}` |
 | `bbb_greenlight_secret` | Secret for greenlight _(required when using greenlight)_ |  | can be generated with `openssl rand -hex 64`
 | `bbb_greenlight_db_password` | Password for greenlight's database  _(required when using greenlight)_ | | can be generated with `openssl rand -hex 16`
 | `bbb_greenlight_default_registration` | Registration option open(default), invite or approval
 | `bbb_disable_recordings` | Disable options in gui to have recordings | `no` | [Recordings are running constantly in background](https://github.com/bigbluebutton/bigbluebutton/issues/9202) which is relevant as privacy relevant user data is stored
 | `bbb_api_demos_enable` | enable installation of the api demos | `no` |
+| `bbb_mute_on_start:` | start with muted mic on join | `no` |
+| `bbb_app_log_level:` | set bigbluebutton log level | `DEBUG` |
```
Comment (n0emis) [inline README.md:28]: You also need to specify this in the `defaults/main.yml`...

Comment (genofire) [reply README.md:28]: ups, thought i have ;(

File: README.md (hunk 1 above)
Comment (n0emis) [reply README.md:15]: No, no quite yet. Take a look at `test/integration/default*/default.yml`

Comment (genofire) [reply README.md:15]: Found it already ;)

**Pull request merged**